packages/esm-patient-list-app/src/create-edit-patient-list/create-edit-list.component.tsx

```diff
@@ -58,6 +58,7 @@ const CreateEditPatientList: React.FC<CreateEditPatientListProps> = ({
           }),
         )
         .then(onSuccess)
+        .then(() => setSubmitting(false))
```

Comment:
```
.then(() => {
onSuccess();
setSubmitting(false);
})
```

Reply:
Not really because the `.catch` is also there, you realize we have another `.then` to close the modal and it closes even if there is error, but this is a better approach, thanks.

Reply:
`.then` is only called when there's a success from the previous async call. If there's any error in any of the chained `.then`, it will directly fall to the `.catch` phrase, and hence it might not run `setSubmitting(false)`

Reply:
Oh I see, it's okay I already used the approach you suggested above.